### DISCUSSION TITLE
How to disable "vscode.ipynb" programmatically

### DISCUSSION THREAD

**Discussion by mcnesium** [microsoft/vscode-jupyter#16164] [Questions and Answers] [answered]
We can disable VSCode's Jupyter Notebook support by searching the extension with the "builtin" filter and the "disable" button like e.g. explained in https://github.com/microsoft/vscode/issues/152007.

However, I want to persist this setting in my [home-manager config](https://home-manager-options.extranix.com/?query=vscode), but I can not find, where and how this setting is written down. How can I disable that extension "programmatically"?

**Comment by DonJayamanne** [comment]
@isidorn @sandy081 /cc

**Reply by isidorn** [reply, marked as answer]
It is in the local storage of vscode.
You can not disable an extension programmatically right now.
